src/Perception/src/ArucoDetector.cpp

```diff
@@ -55,6 +55,11 @@ class ArucoDetector::Impl
     /**
      * Utility map for choosing Aruco marker dictionary depending on user parameter
      */
+    // TODO(traversaro): when we drop support for OpenCV < 4.7.0, we can cleanup this part
+    // we can also check if there are other dictionary that should be added here
+#if (CV_VERSION_MAJOR >= 5) || (CV_VERSION_MAJOR == 4 && CV_VERSION_MINOR >= 7)
```

Comment:
Yes, there are certainly other dictionaries to be added here. I remember experiencing this issue earlier and purposefully leaving out some dictionaries from the lookup.

Apriltags are not added here. https://docs.opencv.org/3.4/dc/df7/dictionary_8hpp.html